contracts/Auction.sol

```diff
@@ -166,9 +150,19 @@ contract Auction is IAuction {
     ///      one is to buy entire auction with the amount desired for this auction.
     ///      The other way is to buy a portion of an auction. In this case an
     ///      auction depleting rate is increased.
-    /// @param amount the amount the taker is paying, denominated in tokenAccepted
-    function takeOffer(uint256 amount) public override {
-        // TODO frontrunning mitigation
+    ///      WARNING: When calling this function directly, it might happen that
+    ///      the expected tokens to seize from the coverage pool is different
```

Comment:
Probably "amount" got lost.
"it might happen that the expected **amount of** tokens"

Reply:
842984e